docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,280 @@
+```
+ 
+Here is our VirtualServer configuration for Istio service mesh: (note `use-cluster-ip` and `requestHeaders`). These settings are required when using Istio service mesh.
+
+```yaml
+apiVersion: k8s.nginx.org/v1    
+kind: VirtualServer    
+metadata:    
+  name: cafe    
+  namespace: nginx-ingress
+spec:    
+  host: cafe.example.com    
+  tls:    
+    secret: cafe-secret    
+  upstreams:    
+  - name: tea    
+    service: tea-svc    
+    port: 80    
+    use-cluster-ip: true
+  - name: coffee    
+    service: coffee-svc    
+    port: 80    
+    use-cluster-ip: true
+  routes:    
+  - path: /tea    
+    action:
+      proxy:
+        upstream: tea
+        requestHeaders:
+          set:
+          - name: Host
+            value: tea-svc.nginx-ingress.svc.cluster.local
+  - path: /coffee
+    action:
+      proxy:
+        upstream: coffee
+        requestHeaders:
+          set:
+          - name: Host
+            value: coffee-svc.nginx-ingress.svc.cluster.local
+```
+
+With our new Host header control in v1.11, when VirtualServer is configured with `requestHeaders`, the value specified will be used and `proxy_set_header $host` will NOT be used.    
+The value of `requestHeaders` should be: `<service.namespace.svc.cluster.local>`. Adjust the value for your specific enviornment.   
+
+When `use-cluster-ip` is set to `true`, NGINX forwards requests to the service IP. In our example above, that would be `tea-svc` and `coffee-svc`.
+
+Here is a simple example of what your `upstream` section will look like now in `virtualServer/virtualServerRoute`:
+
+```yaml
+upstreams:
+  - name: tea
+    service: tea-svc
+    port: 80
+    use-cluster-ip: true
+  - name: coffee
+    service: coffee-svc
+    port: 80
+    use-cluster-ip: true
+```
+
+Now NGINX Ingress `upstreams` will be populated with the `Service/cluster IP`. In the example above, the service/cluster IPs for `tea-svc` and `coffee-svc` will be added to the `upstream` configuration as the `server` addresses.
+
+Now we can test our NGINX Ingress with Istio setup with a simple curl request to our application.    
+
+```bash
+curl -kI https://cafe.example.com/coffee     
+
+HTTP/1.1 200 OK
+Server: nginx/1.19.5
+Date: Thu, 25 Mar 2021 18:47:21 GMT
+Content-Type: text/plain
+Content-Length: 159
+Connection: keep-alive
+expires: Thu, 25 Mar 2021 18:47:20 GMT
+cache-control: no-cache
+x-envoy-upstream-service-time: 0
+x-envoy-decorator-operation: coffee-svc.nginx-ingress.svc.cluster.local:80/*
+```
+
+We can see in the above output that our curl request is sent and received by NGINX Ingress Controller. We can see that the envoy sidecar proxy sends the request to the service IP to the application (coffee). The full request is complete and correct. Now we have a full working NGINX+ Ingress with Istio as the sidecar proxies are deployed.
+
+
+For disabling/removing sidecar proxies and autoinjection:

```

Comment:
is this a section or is this missing steps right under?  This is followed with "To remove label from the namespace:"

Reply:
This is to show that the envoy sidecar proxy is picking up the traffic that is being passed to it by NGINX Ingress controller.
The next section would be a "clean up" phase to remove istio as needed.